src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/VisitResourceIntTest.kt

```diff
@@ -44,9 +49,32 @@ class VisitResourceIntTest : IntegrationTestBase() {
   @Autowired
   lateinit var transactionManager: PlatformTransactionManager
 
+  @Autowired
+  lateinit var repository: Repository
+
+
   @DisplayName("Create")
   @Nested
   inner class CreateVisitRequest {
+    lateinit var offenderNo: String
+    lateinit var offender: Offender
+    private var offenderBookingId: Long = 0L
+
+    @BeforeEach
+    internal fun createPrisoner() {
+      offender = repository.save(
```

Comment:
same offender is created for each test, we could do this per suite if it was too slow but obvious tests might interfere with each other (which in this case they won't)